doc/devdocs/modules/powerpreview/monaco/readme.md

```diff
@@ -0,0 +1,15 @@
+# Developer Preview (Monaco)
+## Update monaco editor
+
```

Comment:
Add description before what Monaco is/does and that it is maintained external by VisualStudio team. (I think not more than three sentences at all.)